example-app/SantokuApp/src/bases/ui/common/getSpacedChildren.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {Box} from './index';
+import {StyledSpacingKeys} from './types';
+
+export const getSpacedChildren = (
+  children: React.ReactNode[] | React.ReactNode,
+  space: StyledSpacingKeys,
+  axis: 'X' | 'Y',
+): any => {
+  let childrenArray = React.Children.toArray(children);
+
+  const spacingProp: object = {
+    ...(axis === 'X' ? {pl: space} : {pt: space}),
+  };
+  childrenArray = childrenArray.map((child, index) => {
```

Comment:
Is `childrenArray` an unnecessary variable?

```
return React.Children.toArray(children).map(() => {....})
```

Reply:
fixed
ae546f1d1c6f1a226739a504cfaa8165d1c6cb6b